datastore/additional_data/sources/codelist_code.py

```diff
@@ -83,11 +98,52 @@ def update_additional_data(self, grant, additional_data):
         except (KeyError, CodelistCode.DoesNotExist):
             pass
 
+        try:
+            code = grant["locationScope"]
+            locationScope = CodelistCode.objects.get(
+                code=code, list_name="locationScope"
+            ).title
+        except (KeyError, CodelistCode.DoesNotExist):
+            pass
+
+        try:
+            for location in grant["beneficiaryLocation"]:
```

Comment:
Should the try block be inside the for loop here, otherwise if the first location can't be found, it won't even try the rest?

Reply:
'cause of `CodelistCode.objects.get` raising

Reply:
yep good point